packages/react/src/components/List/ListContent/ListContent.jsx

```diff
@@ -68,6 +69,12 @@ const propTypes = {
   /** icon can be left or right side of list row primary value */
   iconPosition: PropTypes.oneOf(['left', 'right']),
   selectedItemRef: HtmlElementRefProp,
+  /** does this list use infinite scrolling */
+  isInfiniteScroll: PropTypes.bool,
+  /** callback to fire when the last element in an infinite scroll list is visible */
+  onInfiniteScroll: PropTypes.func,
```

Comment:
what are the params

Reply:
Currently, In this example there aren't any params passed. This was added purely as an example to get your comments and feedback about how we should build this feature. There a more robust example that is built on top of this PR here (https://github.com/carbon-design-system/carbon-addons-iot-react/pull/3140). In that example, I pass `pageSize`, `start`, and `retry` parameters. `pageSize` is the number of items to load, `start` is the starting index to load from, and `retry` is a reference to most recently called attempt to load to handle retry logic.